src/linemod_train.cpp

```diff
@@ -130,46 +134,59 @@ namespace ecto_linemod
 #else
     RendererOSMesa renderer = RendererOSMesa(mesh_path);
 #endif
-    renderer.set_parameters(width, height, focal_length_x, focal_length_y, near,
+    renderer.set_parameters(width, height, double(focal_length_x), double(focal_length_y), near,
         far);
     std::remove(mesh_path.c_str());
 
-      RendererIterator renderer_iterator = RendererIterator(&renderer, 150);
-
-      cv::Mat image, depth, mask;
-      cv::Rect rect;
-      cv::Mat_<unsigned short> depth_short;
-      cv::Matx33d R;
-      cv::Vec3d T;
-      for (size_t i = 0; !renderer_iterator.isDone(); ++i, ++renderer_iterator)
-      {
+    RendererIterator renderer_iterator = RendererIterator(&renderer, 150);
+    //set the RendererIterator parameters
+    renderer_iterator.angle_step_ = 10;
```

Comment:
for now, let them as is in the code. But they should be set per object as, in the future, they will be computed per object.